Tests/Build/Utilities.swift

```diff
@@ -0,0 +1 @@
+../Functional/Utilities.swift
```

Comment:
That' s a link to share `Utilities.swift` file. It has `mktmpdir` and other utility functions.